frontend/src/components/profile/info/badgeHandler.ts

```diff
@@ -217,6 +244,15 @@ function getBadgeGlow(level: number, badge: BadgeType): string | undefined {
     return isMaxBadge(level, badge) ? BADGE_MAX_GLOW_COLOR[badge] : undefined;
 }
 
+/**
+ * Gets the cohort badge glow color
+ * @param cohort the cohort
+ * @returns the glow color
+ */
+function getCohortBadgeGlow(cohort: string): string {
```

Comment:
Why do we need this function if all it does is return the color from the map? Why not just use the map directly?